docker/api/Dockerfile

```
@@ -12,27 +12,30 @@ LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security
 
 ARG CISA_UID=421
 ENV CISA_HOME="/home/cisa"
-ENV ECHO_MESSAGE="Hello World from Dockerfile"
 
 RUN addgroup --system --gid ${CISA_UID} cisa \
   && adduser --system --uid ${CISA_UID} --ingroup cisa cisa
 
-RUN apk --update --no-cache add \
-ca-certificates \
-openssl \
-py-pip
+# RUN apk --update --no-cache add \
+
+RUN apt-get update && \
+ apt-get install wget
```

We should only install what is necessary and remove any apt cruft when we're done installing packages.

```suggestion
RUN apt-get update \
  && apt-get install -y --no-install-recommends \
    wget \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
```